File: TransCoder_tokenized_test_set_functions/java/C_PROGRAM_FIND_AREA_TRIANGLE.java
static float findArea ( float a, float b, float c ) {
  if ( a < 0 || b < 0 || c < 0 || ( a + b <= c ) || a + c <= b || b + c <= a ) {
    System . out . println ( "Not a valid triangle" );
    System . exit ( 0 );
  }
  float s = ( a + b + c ) / 2;
  return ( float ) Math . sqrt ( s * ( s - a ) * ( s - b ) * ( s - c ) );
}

File: TransCoder_tokenized_test_set_functions/cpp/C_PROGRAM_FIND_AREA_TRIANGLE.cpp
float findArea ( float a, float b, float c ) {
  if ( a < 0 || b < 0 || c < 0 || ( a + b <= c ) || a + c <= b || b + c <= a ) {
    cout << "Not a valid trianglen";
    exit ( 0 );
  }
  float s = ( a + b + c ) / 2;
  return sqrt ( s * ( s - a ) * ( s - b ) * ( s - c ) );
}